vagrant-spk

```diff
@@ -356,24 +355,52 @@ def ssh(args):
     call_vagrant_command(sandstorm_dir, "ssh")
 
 def main():
-    operations = {
-            'setupvm': setup_vm,
-            'up': bring_up_vm,
-            'init': init,
-            'dev': dev,
-            'pack': pack,
-            'publish': publish,
-            'halt': halt,
-            'destroy': destroy,
-            'global-status': global_status,
-            'ssh': ssh,
-    }
-    parser = argparse.ArgumentParser(prog=sys.argv[0])
-    parser.add_argument("command", choices=sorted(operations.keys()))
+    operations = (
+        ('setupvm', setup_vm,
+            "Set up a fresh app environment in \n"
+            "  --work-directory (using the provided `stack` \n"
+            "  command arg).  May also init credentials in \n"
+            "  ~/.sandstorm on this host."),
+        ('up', bring_up_vm,
+            "Start the Vagrant VM (and build the VM image \n"
+            "  if necessary)."),
+        ('init', init,
+            "Initialize fresh Sandstorm package definition \n"
+            "  resources in --work-directory."),
+        ('dev', dev, 
+            "Make the current app available to Sandstorm \n"
+            "  in dev mode (after bringing the Vagrant VM `up`)"),
+        ('pack', pack,
+            "Pack the curent app into a \n"
+            "  Sandstorm package."),
+        ('publish', publish, 
+            "Send the current packed app spk to \n"
+            "  the Sandstorm App Market."),
+        ('halt', halt, 
+            "Shut down the Vagrant VM."),
+        ('destroy', destroy, 
+            "Delete the Vagrant VM (but leave \n"
+            "  --work-directory untouched)."),
+        ('global-status', global_status,
+            "Show the state of all active \n"
+            "  Vagrant VMs on this host."),
+        ('ssh', ssh,
+            "SSH into the Vagrant VM."),
+    )
+    op_to_func = dict((cmd, f) for cmd, f, helptext in operations)
+    ops_helptext = '\n'.join(cmd + ': ' + helptext for cmd, f, helptext in operations)
```

Comment:
I think I'd be happier if the help text and the choices list were sorted in the same order, alphabetically.  It's a little bit sad that we lose the "lifecycle ordering" above, but I think consistency is more valuable, and most people will probably use `--help` to remember what we call specific subcommands, rather than to learn the high-level lifecycle concepts, which I think are laid out in more detail in the sandstorm docs.